### ISSUE TITLE
[Report] Games utilizing Proton+Vulkan crash on launch with supported hardware

### ISSUE THREAD

**Issue by sosodev** [holoiso-staging/issuetracker#47]
Note that game's with a native Linux build seem to work fine.

## Your Issue:
```
Running any game that utilizes Proton+Vulkan causes a crash with the text (summarized): "Assertion failed! dlls/winevulkan/loader_thunks.c line: 3226 Expression: !status && vkCreateSwapchainKHR"
```

## Your device's full specs
```
CPU: AMD 5800X3D
GPU: AMD RX6600
```

## Installer used to install?
```
holoiso_rel_snapshot20240603.0905.04-InstallerBuild-20240603104443-x86_64
```

## OS version (/etc/holoiso-release):
```
1.3-mesa-fixes
```

**Comment by ajoy39** [comment]
I am having this issue as well, but only when in gaming mode. On the desktop and in regular big picture mode the games launch, in gaming mode they crash. Hades does not show any error messages when it crashes, just returns to the game launch screen, Kingdom Hearts 1.5+2.5HD shows a dialog box with the wine error OP posted. I can choose OK to exit the program, or Cancel to start the wine debugger, both of which just bring me back to the game launch screen. 

Note that Hades has the option to launch as either DirectX or Vulkan (Steam Deck) but both of them crash. 

System Specs
---
CPU: AMD 5800X
GPU: AMD 6700XT

Installer used
--- 
holoiso_rel_snapshot20240603.0905.04-InstallerBuild-20240603104443-x86_64

OS Version
---
IMAGE_ID=holoiso_rel_snapshot20240603.0905.04
OS_TAG=snapshot20240603.0905.04
RELEASTYPE=rel
ISINTERNAL=no

**Comment by sosodev** [comment]
I installed Bazzite and it works perfectly with a very similar experience to SteamOS. :)

**Comment by joeblack2k** [comment]
Yup same here, downgrade to February iso I still had.. works fine
